boards/panasonic/panb511evb/board.yml

```diff
@@ -7,3 +7,39 @@ board:
     variants:
     - name: xip
       cpucluster: cpuflpr
+    - name: ns
+      cpucluster: cpuapp
+runners:
+  run_once:
+    '--recover':
+    - runners:
+      - nrfjprog
+      - nrfutil
+      run: first
+      groups:
+      - boards:
+        - panb511evb/nrf54l15/cpuapp
+        - panb511evb/nrf54l15/cpuflpr
+        - panb511evb/nrf54l15/cpuflpr/xip
```

Comment:
Add `panb511evb/nrf54l15/cpuapp/ns` as well. Also in `--erase` and `--reset` (see 276d447f511c077a90575dd53aabe82a02f54301).

Reply:
Done